[TPE-2025]  Create Taipei 2025 event

thebrecht opened the pull request:
- Add 2025 event
- Add sponsors

toshywoshy commented on static/img/sponsors/device42.png:
No need for this file, please remove it

thebrecht replied:
File removed as requested.

toshywoshy commented on static/img/sponsors/profitap.png:
No need for this file, please remove it

thebrecht replied:
File removed as requested.

toshywoshy commented on static/img/sponsors/maicoin.png:
Please move to the `assets/sponsors/m/` folder and remove this file

thebrecht replied:
File has been moved to the assets/sponsors/m/  folder as requested.

thebrecht commented:
All requested changes have been completed. I've removed the unnecessary files and moved maicoin.png to the proper location in the assets/sponsors/m/ folder. The changes have been committed to the branch.
